### ISSUE TITLE
healthmgr config health.policy.interval.ms should be a string

### ISSUE THREAD

**Issue by huijunw** [twitter/heron#2225]
in config/xxx/healthmgr.yaml, the example config `health.policy.interval.ms: 120000` should be `health.policy.interval.ms: "120000"` otherwise the exception is thrown

```
[2017-08-21 14:09:42 -0700] [STDERR] stderr: Exception in thread "main"   
[2017-08-21 14:09:42 -0700] [STDERR] stderr: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
  at com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy.<init>(DynamicResourceAllocationPolicy.java:69)
  while locating com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy

1 error  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1025)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1051)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.HealthManager.initializePolicies(HealthManager.java:295)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.HealthManager.initialize(HealthManager.java:279)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.HealthManager.main(HealthManager.java:213)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: Caused by: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy.<init>(DynamicResourceAllocationPolicy.java:78)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.DefaultConstructionProxyFactory$2.newInstance(DefaultConstructionProxyFactory.java:86)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)  
[2017-08-21 14:09:42 -0700] [STDERR] stderr: 	... 4 more 
```

**Comment by ashvina** [comment]
Good catch. The solution is the other way around, numbers need not be specified as strings. I am testing the fix locally. Will create a PR soon.

**Comment by huijunw** [comment]
fixed in #2226
